app/helpers/Constants.js

```diff
@@ -0,0 +1,3 @@
+import { Dimensions } from 'react-native';
+
+export const { width, height } = Dimensions.get('window');
```

Comment:
Dimensions could change:

> Although dimensions are available immediately, they may change (e.g due to device rotation, foldable devices etc) so any rendering logic or styles that depend on these constants should try to call this function on every render, rather than caching the value (for example, using inline styles rather than setting a value in a StyleSheet).

Reply:
Will do it a separate PR, thanks for advice